app/controllers/updates_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UpdatesController < ApplicationController
+  def show
```

Comment:
This should be an index action, it is effectively showing an index of updates, it is more RESTful this way